templates/miq-template.yaml

```diff
@@ -97,6 +131,26 @@ objects:
     resources:
       requests:
         storage: ${APPLICATION_REGION_VOLUME_CAPACITY}
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: "${NAME}-${HTTPD_SERVICE_NAME}"
+  spec:
+    accessModes:
+      - ReadWriteOnce
```

Comment:
What kind of data do we need to save on the PV? Is it configurations? data + configurations? Ideally we would not use PVs to save config data and use configmap/secret to externalize these.